conda/recipes/cudf/recipe.yaml

```diff
@@ -55,6 +55,7 @@ requirements:
     - rapids-build-backend >=0.3.0,<0.4.0.dev0
     - scikit-build-core >=0.10.0
     - dlpack >=0.8,<1.0
+    - numba-cuda >=0.14.0,<0.15.0a0
```

Comment:
Question for our live review session later: does the build-time version of numba-cuda affect the compatible runtime versions? We may want to use `pin_compatible` in the `run:` section to encode that dependency correctly.

We may want to edit `dependencies.yaml` as well depending on the outcome of this discussion.

Reply:
Correct. Keeping them in sync should _always_ work. Keeping a lower bound would in practice probably work forever but probably not ever add anything while exposing us to the small possibility of breaking.